docs/NSR.md

```diff
@@ -1,189 +1,214 @@
+---
+title: Network and Certificate System Security Requirements
+subtitle: Version 1.5
+author:
+  - CA/Browser Forum
+date: 19 October, 2020
+copyright: |
+  Copyright 2020 CA/Browser Forum
 
-**NETWORK AND CERTIFICATE SYSTEM SECURITY REQUIREMENTS** 
+  This work is licensed under the Creative Commons Attribution 4.0 International license.
+---
 
-**Version 1.5 - Effective 19 October 2020**
+# Introduction
 
-\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+## Overview
 
+**Scope and Applicability:**
 
-**Scope and Applicability:**  These Network and Certificate System Security Requirements (Requirements) apply to all publicly trusted Certification Authorities (CAs) and are adopted with the intent that all such CAs and Delegated Third Parties be audited for conformity with these Requirements as soon as they have been incorporated as mandatory requirements (if not already mandatory requirements) in the root embedding program for any major Internet browsing client and that they be incorporated into the WebTrust Service Principles and Criteria for Certification Authorities, ETSI TS 101 456, ETSI TS 102 042 and ETSI EN 319 411-1 including revisions and implementations thereof, including any audit scheme that purports to determine conformity therewith.  In these Requirements, the CA is responsible for all tasks performed by Delegated Third Parties and Trusted Roles, and the CA SHALL define, document, and disclose to its auditors (a) the tasks assigned to Delegated Third Parties or Trusted Roles, and (b) the arrangements made with Delegated Third parties to ensure compliance with these Requirements, and (c) the relevant practices implemented by Delegated Third Parties. 
+These Network and Certificate System Security Requirements (Requirements) apply to all publicly trusted Certification Authorities (CAs) and are adopted with the intent that all such CAs and Delegated Third Parties be audited for conformity with these Requirements as soon as they have been incorporated as mandatory requirements (if not already mandatory requirements) in the root embedding program for any major Internet browsing client and that they be incorporated into the WebTrust Service Principles and Criteria for Certification Authorities, ETSI TS 101 456, ETSI TS 102 042 and ETSI EN 319 411-1 including revisions and implementations thereof, including any audit scheme that purports to determine conformity therewith.  In these Requirements, the CA is responsible for all tasks performed by Delegated Third Parties and Trusted Roles, and the CA SHALL define, document, and disclose to its auditors
+  a.  the tasks assigned to Delegated Third Parties or Trusted Roles, and
+  b.  the arrangements made with Delegated Third parties to ensure compliance with these Requirements, and
+  c.  the relevant practices implemented by Delegated Third Parties.
 
-# 1. GENERAL PROTECTIONS FOR THE NETWORK AND SUPPORTING SYSTEMS
+## Document History
```

Comment:
Thank you.  The fact that the definitions and document history for the NCSSRs are at the end has annoyed me for years.